### PR TITLE
javascript2-homework-week3

### PR DESCRIPTION
**Pull request by qamarfj**

### CODE REVIEW

**Review by aijazraja — COMMENTED** [review]
I could not find any thing to comment
good job

File: javascript/javascript2/week3/presser.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  sCount = 0;
+  lCount = 0;
+  timer.value = "";
+
+  displayError.innerHTML = "";
+  displayLcount.innerHTML = "";
+  displayScount.innerHTML = "";
+  displayWiner.innerHTML = "";
+}
+btnResetGame.addEventListener("click", reset);
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/presser.js]: Once game (countdown )starts make input field readonly and remove eventListener from start button so user cannot enter the time and avoid unusual behavior which triggers once start button get clicked while game is going on.
Its good if user can see countdown .

File: javascript/javascript2/week3/warmup.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  numberOfclick++;
+  function logIfDouble() {
+    if (numberOfclick > 1) {
+      numberOfclic = 0;
+      console.log("double click!");
+    } else numberOfclick = 0;
+  }
+  setTimeout(logIfDouble, 500);
+};
+document.addEventListener("click", logDoubleClick);
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/warmup.js]: Try to rewrite `logDoubleClick` function. It only works for first double click and after that with single click also you can see log message .

**Review by NayanaKamtekar — APPROVED** [review]
Good work!!!
There is a scope for improvement in the fastest presser game.

File: javascript/javascript2/week3/presser.js (hunk 1 above)
Comment (qamarfj) [reply javascript/javascript2/week3/presser.js]: Thank you for your precious time for the review and giving valuable feedback, I have implemented all the changes.

File: javascript/javascript2/week3/warmup.js (hunk 2 above)
Comment (qamarfj) [reply javascript/javascript2/week3/warmup.js]: I have fixed it.

**Pull request merged**